tensorboard/components/tf_ng_tensorboard/core/core.effects.ts

```diff
@@ -53,9 +54,9 @@ export class CoreEffects {
       tap(() => this.store.dispatch(pluginsListingRequested())),
       mergeMap(() => {
         return zip(
-          this.coreService.fetchPluginsListing(),
-          this.coreService.fetchRuns(),
-          this.coreService.fetchEnvironments()
+          this.fetchPluginsListing(),
+          this.interop.fetchRuns(),
+          this.interop.fetchEnvironments()
```

Comment:
When we eventually get rid of the interop, will all these be defined here in core effects?

If so, maybe it makes sense to move them here directly, and instead expose `this.interop.tfStorage`, `this.interop.tfBackend`, `this.interop.paginatedViewStore`?

Reply:
I am very hesitant to expose `tfStorage`, `tfBackend` and `paginatedViewStore` directly to any other part of the application other than what truly needs to know. I've made some change regardless. PTAL

Reply:
Sounds reasonable!